src/regressionmodel.jl

```diff
@@ -1,3 +1,20 @@
+# determine AutoInvLink automatically if a package with an appropriate extension
+# is loaded
+
+"""
+    AutoInvLink
+
+Singleton type indicating that the inverse link should be automatically
+determined from the model type.
+
+This will only work for model types with an appropriate extension on
+Julia 1.9+. If an appropriate extension is not defined, then an error
+will occur.
```

Comment:
```suggestion
!!! compat "Julia 1.9"
    Automatic inverse link determination is implemented using package
    extensions, which are available beginning in Julia 1.9.
    
An error is thrown if the inverse link cannot be determined. This will
always occur with Julia versions prior to 1.9, and will otherwise occur
when no extension has been loaded that specifies the link function for
the model type.
```